plugins/stamina/sh_plugin.lua

```diff
@@ -30,21 +30,10 @@ local function CalcStaminaChange(client)
 		return 0
 	end
 
-	local runSpeed
-
-	if (SERVER) then
-		runSpeed = ix.config.Get("runSpeed") + character:GetAttribute("stm", 0)
```

Comment:
Why?

Removing this removes the slow wind down when you are close to empty on stamina, removing immersion unnecessarily.

Reply:
I thought about this with an older version of helix in mind. My apologies.